packages/turf-boolean-within/index.d.ts

```diff
@@ -3,7 +3,7 @@ import { Feature, GeometryObject } from '@turf/helpers'
 /**
  * http://turfjs.org/docs/#booleanwithin
  */
-export default function (
+export default function booleanWithin(
```

Comment:
Those this actually prevent the Typings from working?

Since it's using `export default` the method name is irrelevant (IMO).

Also, this module will be moving entirely to Typescript, check out https://github.com/Turfjs/turf/pull/1263 for examples.